samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -901,6 +910,14 @@ int link_shell(const struct shell *shell, size_t argc, char **argv)
 				return -EINVAL;
 			}
 			break;
+		case LINK_SHELL_OPT_NCELLMEAS_CONTINUOUS_INTERVAL_TIME:
+			periodic_time = atoi(optarg);
+			if (periodic_time == 0) {
```

Comment:
Maybe it would be better to check that the value is positive.

Reply:
I don't understand how that would be better for this case when using atoi().

Reply:
`atoi()` also parses negative values, so `periodic_time` can be negative.

https://en.cppreference.com/w/c/string/byte/atoi

Reply:
A good point again, thanks. I will change this and couple similars in this file.